weave-js/src/entrypoint.tsx

```diff
@@ -10,6 +10,17 @@ import {WeaveMessage} from './components/Panel2/WeaveMessage';
 import {NotebookComputeGraphContextProvider} from './contextProviders';
 import {URL_LOCAL, URL_RECENT, URL_WANDB} from './urls';
 
+// These get popuated via /__frontend/env.js and are defined in weave_server.py
+declare global {
+  interface Window {
+    CONFIG: {
+      PREFIX: string;
+      ANALYITCS_DISABLED: boolean;
+      WEAVE_BACKEND_HOST: string;
```

Comment:
the `WEAVE_BACKEND_HOST` is statically set to `__weave` ... do we expect the resolve location to be `HOST/PREFIX/__weave` or `HOST/__weave`? If the former, i think we need to update `backendWeaveExecutionUrl` and sister functions to be appropriately prefixed

Reply:
Mentioned in my other reply, we expose `/__weave` and `__frontend` at the root to keep things simpler.  This get's more complex if the server itself is under another prefix but we can wire that up to `WEAVE_BACKEND_HOST`.